conjur/api.py

```diff
@@ -284,18 +284,17 @@ def update_policy_file(self, policy_id, policy_file):
 
         return self._load_policy_file(policy_id, policy_file, HttpVerb.PATCH)
 
-    def rotate_other_api_key(self, resource, resource_to_rotate):
+    def rotate_other_api_key(self, resource_obj):
```

Comment:
does the `obj` suffix is needed?

Reply:
no but think it is clear that an object is needed and not a string for readability purposes but can change this to

`def rotate_other_api_key(self, resource: Resource):`

to achieve this readability

Reply:
You may regret you proposed it...
Yes, this is wonderful! you should do it more across all the project!
You can do this for return value too, so take advantage of this too!
Also, static code analysis should identify this in case you use incorrect argument type.
Do we do this kind of analysis on our code? Will it find this mismatch in case wrong type was passed as arg?
In case it doesn't, you should make sure it does, so you could benefit from using this feature.

Reply:
haha! Sounds like a separate PR. I want to keep this PR smaller but yes will implement!